IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,66 @@
+### IPFS All Hands Format
+
+![](https://media.giphy.com/media/JFawGLFMCJNDi/giphy.gif)
+
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS ON CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use this platform to:
+
+1. Attract skilled contributors to your endeavor
+
+2. Motivate the community about adding support for a new feature
+
+3. Find new partners, capabilities, or challenges for your project to work on/with.
+
+
+The IPFS ALL HANDS ON CALL also gives you a chance to practice public speaking skills. Think of it as a [Toast Masters](https://www.toastmasters.org/) for the IPFS crowd. Below are some public speaking resources.
```

Comment:
I believe the intention was to actually capture more of the content that the IPFS Community has been producing. If the intent is to get people good at public speaking skills, there are better ways to do it.

I particularly don't think that screen-sharing to 50 people + do Live Stream to hundreds of contributors is the best way to practice, specially given that there is no feedback to the talk itself. Toastmasters in the other hand has a very structured process to help people growth on this end.

Reply:
@daviddias for the record, I had also heard the argument that changing to this format would be a great way to give our community a friendly space to test how to communicate new ideas and practice presenting generally. To put it another way, this new structure would allow us to find and encourage people who might not normally engage (to your point about capturing more of what our community has been doing), and a selling point to get people excited to do it would be the friendly presentation space and practice. Perhaps it is not quite Toastmasters, but I don't think the point is like TOTALLY out of line ;)

Reply:
In order to make the All Hands a very useful for a public speaking training, then we can take a lot of inspiration from Toastmasters. In Toastmasters they always have a time keeper, grammarian, "Ah" counter, evaluator and at the end, everyone can give feedback to the talk.

Reply:
To David's point, while I agree we should be a friendly, welcoming venue for all kinds of presenters, I don't think we should be pitching this call as a toastmasters or incorporating toastmaster-style feedback into the format. I think the majority of our community is attending for the content communicated about improvements to IPFS or applications built on top - and don't want to distract from that communication channel with meta-level speaking feedback that might dissuade high-leverage community members from engaging or seeing this call as a valuable space to present their work. 

Perhaps we could rephrase this as "... also gives you a chance to practice communicating your IPFS project or to preview talks for other venues with a friendly and supportive audience."

Reply:
I can see how the association with Toastmasters can be unhelpful and I think the rephrasing _"...also gives you a chance to practice communicating your IPFS project or to preview talks for other venues with a friendly and supportive audience."_ probably is more accurate reflection of how speakers can use IPFS All Hands meeting as a space to practice public speaking.